data/kpi_readme.md

```diff
@@ -72,26 +72,32 @@ Example event data from the `ahoy_visits` table.
+| [x]  | User profile                  | `UserController#show`                    | `profile_page`                  | `/my-account`                                |
+| [x]  | User name change              | `UserController#update_name`             | `user_name_change`              | `/my-account/update-name`                    |
+| [x]  | User email change             | `UserController#update_email`            | `user_email_change`             | `/my-account/update-email`                   |
+| [x]  | User password change          | `UserController#update_password`         | `user_password_change`          | `/my-account/update-password`                |
+| [x]  | User postcode change          | `UserController#update_postcode`         | `user_postcode_change`          | `/my-account/update-postcode`                |
+| [x]  | User ofsted change            | `UserController#update_ofsted_number`    | `user_ofsted_change`            | `/my-account/update-ofsted-number`           |
+| [x]  | Email address taken           | `RegistrationsController#create`         | `email_address_taken`           | `/users/sign-up`                             |
+| [x]  | User inactivity logout        | `ErrorsController#timeout`               | `error_page`                    | `/timeout`                                   |
+| [x]  | 404 Error                     | `ErrorsController#not_found`             | `error_page`                    | `/404`                                       |
+| [x]  | 500 Error                     | `ErrorsController#internal_server_error` | `error_page`                    | `/500`                                       |
+| [x]  | Module start                  | `ContentPagesController#show`            | `module_start`                  | `/modules/{alpha}/content-pages/{intro}`     |
+| [x]  | Module complete               | `TrainingModulesController#certificate`  | `module_complete`               | `/modules/{alpha}/certificate`               |
+| [x]  | Questionnaire answered        | `QuestionnairesController`               | `questionnaire_answer`          | `/modules/{alpha}/questionnaires/{path}`     |
+| [x]  | Summative assessment start    | `QuestionnairesController#show`          | `summative_assessment_start`    | `/modules/{alpha}/questionnaires/{path}`     |
+| [x]  | Summative assessment complete | `AssessmentResultsController#show`       | `summative_assessment_complete` | `/modules/{alpha}/assessment-results/{path}` |
+| [x]  | Confidence check start        | `QuestionnairesController#show`          | `confidence_check_start`        | `/modules/{alpha}/questionnaires/{path}`     |
+| [x]  | Confidence check complete     | `ContentPagesController#show`            | `confidence_check_complete`     | `/modules/{alpha}/questionnaires/{path}`     |
 
 
 ## To be completed
```

Comment:
We should just delete the whole "to be completed" section when it is done and make sure the table reflects the changes coming in the other PR.